src/components/List.js

```diff
@@ -79,6 +80,21 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  // Confirm that the user would like to delete this item
+  const confirmDelete = (item) => {
+    const confirmed = window.confirm(
+      'Are you sure you want to delete this item?',
+    );
+    if (confirmed) {
+      try {
+        db.collection(token).doc(item.id).delete();
+      } catch (error) {
+        setDeleteError(`Problem deleting ${item.itemName}`);
+        console.log(error);
```

Comment:
If you want to log out an error, I'd recommend using `console.error` so that it clearly shows up as an error in the console.